tests/test_expectation_maximization.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+
+from iterative_refinement import expectation_maximization
+
+
+def test_split_array():
+    arr = np.zeros(4)
```

Comment:
instead of hardcoding all these shapes I suggest you choose a reasonable size globally, e.g. `n_pix = 128`. Better yet each time you test generate a random even integer, ranging 100-200.

See 
https://github.com/compSPI/simSPI/blob/master/tests/test_transfer.py#L10

Reply:
will do